### DISCUSSION TITLE
SentMessageInfo of @types/nodemailer missing properties

### DISCUSSION THREAD

**Discussion by Whobeu** [DefinitelyTyped/DefinitelyTyped#54161] [Issues with a @types package]
In @types/nodemailer 6.4.0 and earlier, the info callback argument of sendMail was defined as any. However, with 6.4.2, the argument is defined as type SentMessageInfo. The problem is SentMessagInfo is missing the accepted, rejected, pending and response properties. Basically the SentMessageInfo interface needs to have these four additional properties added based on the documentation at [https://nodemailer.com/usage/#sending-mail](https://nodemailer.com/usage/#sending-mail)

```
    interface SentMessageInfo {
        /** includes the envelope object for the message */
        envelope: MimeNode.Envelope;
        /** most transports should return the final Message-Id value used with this property */
        messageId: string;
        accepted: Array<string | Mail.Address>;
        rejected: Array<string | Mail.Address>;
        pending: Array<string | Mail.Address>;
        response: string;
    }

```

**Comment by Whobeu** [comment]
https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/54202

**Comment by samarmohan** [comment]
Do you want to pr this or should I?

**Reply by Whobeu** [reply]
I've never gone through a formal fix process before so if you want to handle it that is fine by me. The modified SentMessageInfo interface with the four extra types works in my testing so there should not be any issues.

**Reply by samarmohan** [reply]
Cool, thanks! I’ll get a pr started in a few hours.
